test/modelling/space_shuttle_reentry_trajectory.jl

```diff
@@ -0,0 +1,179 @@
+
+using Interpolations, Plots
+using JuMP, Ipopt  # , KNITRO
+
+
+# Global variables
+const w  = 203000  # weight (lb)
+const g₀ = 32.174  # acceleration (ft/sec^2)
+const m  = w / g₀  # mass (slug)
+
+# Aerodynamic and atmospheric forces on the vehicle
+const ρ₀ =  0.002378
+const hᵣ =  23800.0
+const Rₑ =  20902900.0
+const μ  =  0.14076539e17
+const S  =  2690.0
+const a₀ = -0.20704
+const a₁ =  0.029244
+const b₀ =  0.07854
+const b₁ = -0.61592e-2
+const b₂ =  0.621408e-3
+const c₀ =  1.0672181
+const c₁ = -0.19213774e-1
+const c₂ =  0.21286289e-3
+const c₃ = -0.10117249e-5
+
+
+user_options_ipopt = (
+    "mu_strategy" => "monotone",
+    "linear_solver" => "ma27",
```

Comment:
By setting this option, I am wondering if Ipopt fallbacks to Mumps if not compiled with `ma27`. I would maybe add a comment to explain that.

Reply:
Something like?
```
"linear_solver" => "ma27",  # Ipopt may fallback to Mumps, if not compiled with ma27.
```

Reply:
In that line. I would maybe precise that MA27 is generally faster than MUMPS (a lot of people using Ipopt do not know that we could change the underlying linear solver)